### DISCUSSION TITLE
Tsunami Warning Mock

### DISCUSSION THREAD

**Discussion by LessThan3Chips** [xspanger3770/GlobalQuake#248] [Show and tell]
This wouldn't cause a tsunami warning, but just wanted to test how it'd look in my opinion.
![Untitled](https://github.com/xspanger3770/GlobalQuake/assets/65959770/105c8e48-6594-40ad-8e5f-b922ff406b26)

**Comment by Alemen1190** [comment]
Hello, check how you did to get the tsunami alert
